src/native/libs/build-local.sh

```diff
@@ -13,6 +13,26 @@
 SHIM_SOURCE_DIR="$1"/native/src
 INTERMEDIATE_OUTPUT_PATH="$2"
 TARGET_LIBRARY="$3"
+EMBED_ICU_DATA_PATH="$4"
+
+if [ -n "$EMBED_ICU_DATA_PATH" ]; then
+    header="${INTERMEDIATE_OUTPUT_PATH}embedded_icu_data.h"
+
+    cat <<EOF > "$header"
+// <auto-generated/>
```

Comment:
Is there any prior art for marking auto-generated C/C++ files with `// <auto-generated/>`? I thought that it is a C#-ism.

Reply:
_version.c is using a full license https://github.com/dotnet/runtime/blob/365dde344601cc5310493133cfc1ee7588be1c6f/eng/native/version/_version.c#L1 we can use the same or just remove the header?

Reply:
```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
```